README.md

```diff
@@ -6,26 +6,29 @@
-It provides a straightforward way to boost your application's performance by storing **fragments** of server-side rendered pages in **edge cache servers**.
-It means that after the first rendering, fragments of your pages will be served in a few milliseconds by servers close to your end users!
+
+It provides a straightforward way to boost your application's performance by storing **fragments** of server-side rendered pages in **edge cache servers**.  
+It means that after the first rendering, fragments of your pages will be served in a few milliseconds by servers close to your end users!  
 It's a very efficient way to improve the performance and the SEO of your websites and to dramatically reduce both your hosting costs and the energy consumption of these applications. Help the planet, use React ESI!
 
-Because it is built on top of the [Edge Side Includes (ESI)](https://www.w3.org/TR/esi-lang) W3C specification, 
-React ESI natively supports most of the well-known cloud cache providers including [Cloudflare Workers](https://blog.cloudflare.com/edge-side-includes-with-cloudflare-workers/), [Akamai](https://www.akamai.com/us/en/support/esi.jsp) and [Fastly](https://docs.fastly.com/guides/performance-tuning/using-edge-side-includes).
+Because it is built on top of the [Edge Side Includes (ESI)](https://www.w3.org/TR/esi-lang) W3C specification,
+React ESI natively supports most of the well-known cloud cache providers including [Cloudflare Workers](https://blog.cloudflare.com/edge-side-includes-with-cloudflare-workers/), [Akamai](https://www.akamai.com/us/en/support/esi.jsp) and [Fastly](https://docs.fastly.com/guides/performance-tuning/using-edge-side-includes).  
 Of course, React ESI also supports the open source [Varnish cache server](https://varnish-cache.org/intro/index.html#intro) that you can use in your own infrastructure for free ([configuration example](https://github.com/zeit/next.js/blob/canary/examples/with-react-esi/docker/varnish/default.vcl)).
 
-Also, React ESI allows to specify of different Time To Live (TTL) per React component and generating the corresponding HTML asynchronously using a secure (signed) URL.
-The cache server fetches and stores in the cache all the needed fragments (the HTML corresponding to every React component), builds the final page, and sends it to the browser.
+Also, React ESI allows to specify of different Time To Live (TTL) per React component and generating the corresponding HTML asynchronously using a secure (signed) URL.  
+The cache server fetches and stores in the cache all the needed fragments (the HTML corresponding to every React component), builds the final page, and sends it to the browser.  
 React ESI also allows components to (re-)render client-side without any specific configuration.
 
 ![ESI example](https://book.varnish-software.com/4.0/_images/esi.png)
+
 > Schema from [The Varnish Book](https://book.varnish-software.com/4.0/chapters/Content_Composition.html)
```

Comment:
```suggestion
```

This image is in 404. It was:

![image](https://github.com/dunglas/react-esi/assets/57224/d94cca77-c191-402d-8249-845a9572251f)


Maybe @ginifizz can create a new beautiful schema for us?

Reply:
In the meantime, maybe can we just import this image in the repo.

Reply:
Found it https://github.com/varnish/Varnish-Book/blob/master/ui/img/esi.png
```suggestion
![ESI example](https://github.com/varnish/Varnish-Book/blob/master/ui/img/esi.png)

> Schema from [The Varnish Book](https://info.varnish-software.com/resources/varnish-6-by-example-book)
```

Reply:
Cool! The link to the file itself is https://raw.githubusercontent.com/varnish/Varnish-Book/master/ui/img/esi.png actually.